src/components/CallView/BottomBar.vue

```diff
@@ -42,7 +39,7 @@ const store = useStore()
 const token = useGetToken()
 const actorStore = useActorStore()
 const breakoutRoomsStore = useBreakoutRoomsStore()
-const isFullscreen = useDocumentFullscreen()
+const isFullscreen = !isSidebar ? useDocumentFullscreen() : ref(false)
```

Comment:
Maybe it would be better to not make it a ref, if not needed:

```js
import { toValue } from 'vue'

const isFullscreen = !isSidebar && useDocumentFullscreen() (true / ref(true))

const fullscreenLabel = computed(() => {
	return toValue(isFullscreen)
```